doc/api/crypto.md

```diff
@@ -537,6 +537,9 @@ of the ciphertext in bytes. See [CCM mode][].
 
 The `decipher.setAAD()` method must be called before [`decipher.update()`][].
 
+When passing a string as the `buffer`, please consider
+[Caveats when using strings as inputs to cryptographic APIs][].
```

Comment:
Non-blocking, but I'd expect it to be this here and elsewhere unless the text is the exact same as the heading section (which I don't think it needs to be).

```suggestion
[caveats when using strings as inputs to cryptographic APIs][].
```